03-setup-ingress-controller/ambassador.md

```diff
@@ -564,22 +667,49 @@ To enable proxy protocol for `AES Backend Services`, you need to run the below s
 4. Save the values file and apply changes using `Helm`:
 
     ```shell
-    HELM_CHART_VERSION="6.9.3"
+    HELM_CHART_VERSION="7.2.2"
 
-    helm upgrade ambassador datawire/ambassador --version "$HELM_CHART_VERSION" \
+    helm upgrade edge-stack datawire/edge-stack --version "$HELM_CHART_VERSION" \
     --namespace ambassador  \
     -f "03-setup-ingress-controller/assets/manifests/ambassador-values-v${HELM_CHART_VERSION}.yaml"
     ```
 
-5. Enable the proxy support in the `Ambassador` stack via the [aes_proxy_module](/03-setup-ingress-controller/assets/manifests/ambassador/aes_proxy_module.yaml) manifest.
+5. Open and inspect the `03-setup-ingress-controller/assets/manifests/ambassador/ambassador_listener.yaml` file provided in the `Starter Kit` repository, using an editor of your choice (preferably with `YAML` lint support). For example, you can use [VS Code](https://code.visualstudio.com):
 
     ```shell
-    kubectl apply -f 03-setup-ingress-controller/assets/manifests/ambassador/aes_proxy_module.yaml
+    code 03-setup-ingress-controller/assets/manifests/ambassador/ambassador_listener.yaml
     ```
 
-Please note that module configuration is a `global` option (enable/disable) for `AES`.
+6. Under `spec` section you will need to comment the `protocol` and unncomment the `protocolStack` for both `listener`s, like in the below example:
```

Comment:
I would reformulate this sentence like below:

6. In the `spec` section, you will need to comment out the `protocol` field and enable the `protocolStack` for both `listeners`, like in the example below: